app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :trackable, :lockable, :timeoutable and :omniauthable
+  extend Devise::Models
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  include DeviseTokenAuth::Concerns::User
```

Comment:
please validate the presence of `first_name` and `last_name`. https://guides.rubyonrails.org/active_record_validations.html

Reply:
Done